toolchains/sysimage/build_ext4_image.py

```diff
@@ -91,10 +92,16 @@ def strip_files(fs_basedir, fakeroot_statefile, strip_paths):
         if os.path.isdir(target_path):
             for entry in os.listdir(target_path):
                 del_path = os.path.join(target_path, entry)
-                subprocess.run(["fakeroot", "-s", fakeroot_statefile, "-i", fakeroot_statefile, "rm", "-rf", del_path])
+                flattened_paths.append(del_path)
         else:
-            subprocess.run(["fakeroot", "-s", fakeroot_statefile, "-i", fakeroot_statefile, "rm", "-rf", target_path])
+            flattened_paths.append(target_path)
 
+    if not flattened_paths:
+        return
+
+    subprocess.run(
+        ["fakeroot", "-s", fakeroot_statefile, "-i", fakeroot_statefile, "rm", "-rf"] + flattened_paths,
```

Comment:
is there a limit to how many arguments the rm command will take?

Reply:
I added batching to avoid too many arguments.